proposed/MPID Messaging.md

```diff
@@ -256,7 +244,7 @@ The various different types for `StructuredData::data` can be enumerated as:
 #[allow(variant_size_differences)]
 enum MpidMessageWrapper {
     /// Notification that the MPID Client has just connected to the network
-    Online,
+    Online(mpid_name: NameType),
```

Comment:
I don't think this needs the `mpid_name` field since it will be the sender won't it?